src/MShop.php

```diff
@@ -73,8 +73,12 @@ public static function create( \Aimeos\MShop\ContextIface $context,
 		if( empty( $name ) )
 		{
 			$subpath = !empty( $parts ) ? join( '/', $parts ) . '/' : '';
-			$name = $config->get( 'mshop/' . $domain . '/manager/' . $subpath . 'name' );
-			$name = $name ?: $config->get( 'mshop/' . $domain . '/manager/name', 'Standard' );
```

Comment:
If a name is passed as argument to that method, it's not used any more in your code